test/e2e/node/kubelet.go

```diff
@@ -257,14 +257,6 @@ var _ = SIGDescribe("kubelet", func() {
 			nodeLabels      map[string]string
 			resourceMonitor *e2ekubelet.ResourceMonitor
 		)
-		type DeleteTest struct {
-			podsPerNode int
-			timeout     time.Duration
-		}
-
-		deleteTests := []DeleteTest{
```

Comment:
I'm a little confused about why we got rid of the `DeleteTest` struct. In my mind, it's useful because it allows us to see at a glance the test cases we are examining. In addition, it provides flexibility to easily add more tests.